src/main/java/chatbot/Parser.java

```diff
@@ -0,0 +1,205 @@
+        message.append(t.toString()).append("\n"); // Append the toString representation of t and a newline
+        message
+          .append("Now you have ")
+          .append(myList.len())
+          .append(" tasks in the list."); // Append the third line
+
+        String finalMessage = message.toString();
+
+        return finalMessage;
+      } else if (words[0].equals("deadline")) {
+        String[] parts = detail.split("\\s*/by\\s*", 2);
+        Deadline t = new Deadline(parts[0], parts[1]);
+        myList.add(t);
+        st.rewriteFile(myList.mine());
+        StringBuilder message = new StringBuilder();
+        message.append("Got it. I've added this task:\n"); // Append the first line
+        message.append(t.toString()).append("\n"); // Append the toString representation of t and a newline
+        message
+          .append("Now you have ")
+          .append(myList.len())
+          .append(" tasks in the list."); // Append the third line
+
+        String finalMessage = message.toString();
+
+        return finalMessage;
+      } else if (words[0].equals("event")) {
+        String[] firstSplit = detail.split("\\s*/from\\s*", 2);
+        String[] secondSplit = firstSplit[1].split("\\s*/to\\s*", 2);
+        Event t = new Event(firstSplit[0], secondSplit[0], secondSplit[1]);
+        myList.add(t);
+        st.rewriteFile(myList.mine());
+        StringBuilder message = new StringBuilder();
+        message.append("Got it. I've added this task:\n"); // Append the first line
+        message.append(t.toString()).append("\n"); // Append the toString representation of t and a newline
+        message
+          .append("Now you have ")
+          .append(myList.len())
+          .append(" tasks in the list."); // Append the third line
+
+        String finalMessage = message.toString();
+
+        return finalMessage;
+      } else if (words[0].equals("delete")) {
+        int c = Integer.parseInt(words[1]);
+        Task t = myList.get(c - 1);
+        myList.remove(c - 1);
+        st.rewriteFile(myList.mine());
+        StringBuilder message = new StringBuilder();
+        message.append("Noted. I've removed this task:\n"); // Append the first line
+        message.append(t.toString()).append("\n"); // Append the toString representation of t and a newline
+        message
+          .append("Now you have ")
+          .append(myList.len())
+          .append(" tasks in the list."); // Append the third line
+
+        String finalMessage = message.toString(); // Convert StringBuilder to String
+
+        return finalMessage;
+      } else if (words[0].equals("find")) {
+        int count = 0;
+        StringBuilder result = new StringBuilder();
+
+        for (Task task : myList.mine()) {
+          if (task.getDescription().contains(detail)) {
+            count++;
+            if (count == 1) {
+              result.append(
+                "____________________________________________________________\n"
```

Comment:
Just a suggestion maybe you can include a static final field to wrap this String? Since you are using this multiple times below. maybe your parser class can have a 
private static final String name = "____________________________________________________________\n"